Makefile

```diff
@@ -598,15 +600,41 @@ ifdef PIP_VERSION
 PIP_THEMIS_INSTALL := $(shell pip freeze |grep themis)
 endif
 
-pythemis_install: CMD = cd src/wrappers/themis/python/ && python3 setup.py install --record files3.txt
-pythemis_install:
+pythemis_old_install: CMD = cd src/wrappers/themis/python/ && python3 setup.py install --record files3.txt
```

Comment:
with this change we will break even our old CI/CD scripts. If we want to remove it and migrate to another approach, we should leave previous targets, notify user's who will use it with next themis version that it's deprecate and address them to the new approach. and after one more release we are free to remove it. And we will have time to update our scripts.

Reply:
OK, gonna name it like before. BTW, what do you think about names of all the new `make` targets?